src/internal/symbol/observable.ts

```diff
@@ -1,2 +1,11 @@
+/** Symbol.observable addition */
+/* Note: This will add Symbol.observable globally for all TypeScript users,
+  however, we are no longer polyfilling Symbol.observable */
+declare global {
+  interface SymbolConstructor {
+    readonly observable: symbol;
+  }
+}
+
```

Comment:
I don't think we should revert the moving of this declaration, as its being moved solved a problem. See this PR that made the same change to the 6.x branch and the issue that it references: https://github.com/ReactiveX/rxjs/pull/6178

Reply:
I cherrypicked specific PR https://github.com/ReactiveX/rxjs/pull/6224/commits/ed34f007a7ba76ea6d1383ca50a6f62aa61e29f4